src/components/UserStarsRepo.scss

```diff
@@ -32,7 +32,7 @@
     border-style: solid;
     width: 180px;
     height: auto;
-    background-color: rgba($color: #232529, $alpha: 0.8);
+    background-color: rgba($color: #617aac, $alpha: 0.8);
```

Comment:
I'm not sure if this works, and if it does I'd like to know how

Reply:
background-color modified